README.md

```diff
@@ -8,11 +8,16 @@ You may find some of them opinionated, so we recommend picking only those fittin
 composer require shipmonk/phpstan-rules
 ```
 
-## Rules:
+For a quick start, you can enable all generally usable rules by:
+```neon
+includes:
+    - vendor/shipmonk/phpstan-rules/strict-rules.neon
+```
 
-All you need to enable most of the rules is to register them [as documented in phpstan/phpstan](https://phpstan.org/developing-extensions/rules#registering-the-rule-in-the-configuration).
-Some of them need some specific [rich parser node visitor](https://phpstan.org/blog/preprocessing-ast-for-custom-rules) to be registered as well.
-Rarely, some rules are reliable only when some other rule is enabled.
+Please be aware that some rules are configurable and the default setup used in that file might not fit your needs.
+This repository contains several more rules not included in `strict-rules.neon` which are more opinionated or need custom setup - those are marked by `*`.
```

Comment:
Welcomed change, I'd just consider categorization into 2 sections instead as I find it a bit difficult to make sure that my project uses all the optional rules with current documentation.

Reply:
Agreed to go with this version, we may restructuralize the readme any time in future, this is mainly about existence of the config file and the concept.